<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSeparation</class>
 <widget class="QWidget" name="PlotSeparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gLayPlot"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tBtnStandardCurve">
       <property name="text">
        <string>standard curve</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gLayProportion"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
